### DISCUSSION TITLE
Kerberos Authentication?

### DISCUSSION THREAD

**Discussion by queztaho** [Ylianst/MeshCentral#5303] [Q&A]
A little background: I'm a budding Linux admin but fairly well versed in Windows, but I'm diving further into infrastructure and whatnot, so this is a bit fresh for me in multiple ways.

My question is simple - if it's possible to integrate Kerberos authentication, which is the primary method my institution uses for both local and of course network account authentication for Linux workstations.

LDAP is available as we have a full Active Directory schema deployed, but, Kerberos would be the preferred method for our Linux workstation management. 

Thanks in advance for any insight!

**Comment by si458** [comment]
yes ldap is possible!
https://ylianst.github.io/MeshCentral/meshcentral/#ldap
the is also a youtube video you can watch how to setup ldap etc

**Comment by queztaho** [comment]
Thank you - I did see the LDAP information, I was just hoping that maybe Kerberos would be as/simpler to implement, hence my question. But kerberos isn't _required_  so if it's a bit too much to implement, I'll just fallback to LDAP.
